schemas/2017-06-01/Microsoft.AAD.json

```diff
@@ -0,0 +1,239 @@
+{
+  "id": "https://schema.management.azure.com/schemas/2017-06-01/Microsoft.AAD.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.AAD",
+  "description": "Microsoft AAD Resource Types",
+  "resourceDefinitions": {
+    "domainServices": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.AAD/domainServices"
+          ]
+        },
+        "apiVersion": {
+          "type": "string",
+          "enum": [
+            "2017-06-01"
+          ]
+        },
+        "domainName": {
```

Comment:
>domainName [](start = 9, length = 10)

Is this correct? Isn't this supposed to be inside the "properties" bag? Only "name", "type", "apiVersion" and a few others are allowed at top level.

Reply:
I have fixed this.